__tests__/api.test.js

```diff
@@ -268,3 +291,44 @@ describe("/api/reviews get request to get ALL review objects", () => {
       });
   });
 });
+
+////////////////////// FETCH COMMENTS //////////////////////////////
+
+describe("/api/:review_id/comments get request for comments associated with review via review id", () => {
+  it("return an array of objects", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toEqual(true);
+        expect(typeof body[0] === "object").toEqual(true);
+      });
+  });
+
+  it("ensure each object returned has the requested keys", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((item) => {
+          expect(item.comments).hasOwnProperty("comment_id");
+          expect(item.comments).hasOwnProperty("votes");
+          expect(item.comments).hasOwnProperty("created_at");
+          expect(item.comments).hasOwnProperty("author");
+          expect(item.comments).hasOwnProperty("body");
+          expect(item.comments).hasOwnProperty("review_id");
+        });
+      });
+  });
+
+  it("return an error if no comments exist for that entry", () => {
```

Comment:
Shouldn't be throwing an error if there are no comments. No comments should still be a 200 status code.